.github/ISSUE_TEMPLATE/10-feature-issue.md

```diff
@@ -0,0 +1,33 @@
+If the feature is related to a specific library below, please raise an issue in the respective repo directly:
+
+[TensorFlow Data Validation Repo](https://github.com/tensorflow/data-validation/issues)
+
+[TensorFlow Model Analysis Repo](https://github.com/tensorflow/model-analysis/issues)
+
+[TensorFlow Transform Repo](https://github.com/tensorflow/transform/issues)
+
+[TensorFlow Serving Repo](https://github.com/tensorflow/serving/issues)
+
+
+<em>Please make sure that this is a bug. As per our
+[GitHub Policy](https://github.com/tensorflow/tensorflow/blob/master/ISSUES.md),
+we only address code/doc bugs, performance issues, feature requests and
+build/installation issues on GitHub. tag:bug_template</em>
+
+**System information**
+- TensorFlow version (you are using):
```

Comment:
nit: this may not be very applicable (similar for environment)

Reply:
Removed the line corresponding to Tensorflow version. Please confirm if we have to remove line corresponding to Environment as well?